src/brasil/gov/tiles/tiles/banner_rotativo.py

```diff
@@ -131,7 +71,11 @@ def thumbnail(self, item):
     @view.memoize
     def accepted_ct(self):
         results = ListTile.accepted_ct(self)
-        results.append(u'ExternalContent')
+        try:
```

Comment:
same comment here, but in this case I would initialize either an empty list or a list containing just the element `['ExternalContent']` and I would use `extend()` instead of `append()`.